docs/src/main/asciidoc/cache.adoc

```diff
@@ -682,7 +682,9 @@ Here's what your cache configuration could look like:
 quarkus.cache.caffeine."foo".initial-capacity=10 <1>
 quarkus.cache.caffeine."foo".maximum-size=20
 quarkus.cache.caffeine."foo".expire-after-write=60S
+quarkus.cache.caffeine."foo".cacheNullValues=true
```

Comment:
I'm not sure the default behavior needs to be included here.
```suggestion
```

Reply:
On the other hand, there's a documentation section about caching null values that could use an update with this PR: https://quarkus.io/guides/cache#negative-cache